### ISSUE TITLE
Upgrade fine-uploader

### ISSUE THREAD

**Issue by axelstudios** [SEED-platform/seed#876]
In addition to not using extremely-outdated code, fine-uploader is a pain because it needs to be built by the user with grunt.  Once compatibility with the latest fine-uploader has been added, we can use a pre-built repo like this (https://github.com/feryardiant/fineuploader-dist).

Alternatively, we can add a distributable 3.x fine-uploader build to a new SEED-platform repo and use that.

**Comment by rebrownlbnl** [comment]
so the user would install from that fineuploader-dist repo?
I don't think we can distribute it through SEED-platform because of the license type.

**Comment by axelstudios** [comment]
The dist repo just removes a few dependencies and the step of developers compiling the code themselves.

I didn't notice that the license had changed - v3.9.1 (currently used by seed, circa 2013-10) appears to be GPLv3, and v5.0.9 was the last version with the GPL license before they switched to their custom Widen license.  Their Widen license allows open-source projects to use the code freely, but if that conflicts with the seed license we can either use [prebuilt v5.0.9](https://github.com/feryardiant/fineuploader-dist/releases/tag/5.0.9) or go with the other option I mentioned of pre-building v3.9.1.

**Comment by RDmitchell** [comment]
I don't believe we can use GPL. I think the only license we can use are BSD and MIT. @rebrownlbnl can verify.

**Comment by rebrownlbnl** [comment]
yes, that's my understanding also, which is why we haven't been distributing fineuploader but making the user download it directly from the other source.
If the dist repo just makes it easier to install, but the source is still that other repo, let's go with that.

**Comment by rnicholus** [comment]
Hey there! If you have any licensing questions or anything else related to using Fine Uploader in your project, feel free to let me know or open up a new case in the Fine Uploader issue tracker.

Note that Fine Uploader is also distributed on npm.

**Comment by rebrownlbnl** [comment]
Thanks @rnicholus for offering the assistance.
The npm option may help, although the fineuploader npm page says it's licensed under the commercial license? 
@axelstudios does npm solve any of the issues you are trying to address?

**Comment by rnicholus** [comment]
The widen commercial license allows free projects to use Fine Uploader for free.

**Comment by rnicholus** [comment]
Please note that, as of version 5.9.0, Fine Uploader is now MIT licensed.

**Comment by axelstudios** [comment]
Thanks, @rnicholus!  That's fantastic news

**Comment by rebrownlbnl** [comment]
@rnicholus thanks for the update -- that is very good news.
@axelstudios- given that we can distribute the v5.9.0 code, what's the suggested upgrade path?
